discord-haskell.cabal

```diff
@@ -97,7 +97,7 @@ executable state-counter
                      , discord-haskell
 
 library
-  ghc-options:         -Wall
+  ghc-options:         -Wall -j10
```

Comment:
`-j10` while speeding up compilation also increases the already significant memory usage of GHC substantially and could lead to issues in memory constrained environments.
So I'm not sure it may be a good idea to use